scripts/message/MessageForm.js

```diff
@@ -13,34 +13,79 @@ to save messages sent through the form
         // will trigger the saveMessage function, which will retrieve the values from user inputs and post them to the API
         // return an alert: your message has been sent!
         // once the alert is X'd out, setView function triggers, which will change the "view" property to default view / main page
-// declare and export MessageForm function to build HTML for message form
-
-import { getChosenUser, setView, saveMessage , fetchUsers} from "../data/TransientState.js"
-
-const chosenUsers = getChosenUser()
-// const savedMessages = saveMessage() This function should only be called in the event listener -Ryan
-// const preferredView = setView() This function should only be called in the event listener -Ryan
-
-export const MessageForm = async () => {
-    const users = await fetchUsers()
-
-    let html = `<h1>Create Your Message</h1> 
-   <section>
-   To:
-    <select id="messageRecipientDropdown"> 
-    <option value="0">List of Users`
-    for (const user of users) {
-      html +=  `<option value = "${user.id}">${user.name}</option>`
-    }
-
-return html += `</select></section>
-<section>
-<label for="username">Name:</label><br>
-<input type="text" id="name"><br>
-<label for="text">Subject:</label><br>
-<input type="text" id="subject"><br>
-<label for="message">Message:</label><br>
-<input type="text" id="message"><br>
-<button type="button">Send</button>
-</section>
-` }
+
+        
+
+
+
+        import { getChosenUser, setMessage, setView, saveMessage , fetchUsers, saveLike, getCurrentUser} from "../data/TransientState.js"
+
+
+
+        // const documentingSetMessage = (event) => {
+        //     if(clickEvent.target.id === "send") {
+        //     setMessage(parseInt(clickEvent.target.value))
+        // }
+        // }
+        const sendMessageButtonLink = async (clickEvent) => {
+             if(clickEvent.target.id === "sendMessageForm") {
+                let message = document.getElementById("messageForm").value
+                let currentUser = getCurrentUser()
+                let recipientId = document.getElementById("chooseRecipient").value
```

Comment:
Change `"chooseRecipient"` to `"messageRecipientDropdown"`
We want to target the value whole select element, and then get the 'values' will come from each option